I650/i650_sys.c

```diff
@@ -705,25 +705,49 @@ int Shift_Digits(t_int64 * d, int nDigits)
            
 */
 
-// load card file fn and add its cards to 
-// DeckImage array, up to a max of nMaxCards
-// increment nCards with the number of added cards
-// uses cdr0 device/unit
-t_stat deck_load(CONST char *fn, uint16 * DeckImage, int * nCards)
+/*
+ * Allocate space for a card deck of nCards cards.
+ */
+uint16 *deck_alloc(int nCards)
+{
+    return malloc(nCards * 80 * sizeof(uint16));
+}
+
+/*
+ * Free a deck previously obtained from deck_alloc().
+ */
+uint16 *deck_free(uint16 *deck)
+{
+    free(deck);
+}

```

Comment:
On my Mac, "make i650" fails with an error here, saying that it's a non-void function lacking a return statement.

Presumably the function type should have a void return rather than a pointer.

Reply:
Yes, you're right... just a second...